config/dictionaries/resource.json

```diff
@@ -670,6 +670,29 @@
       }
     ]
   },
+  {
+    "id": "balenaos_versions",
+    "name": "List balenaOS versions",
+    "fields": [],
+    "examples": [
+      {
+        "id": "list-balenaos-versions-for-private-device-type",
+        "summary": "List the supported balenaOS versions for a private device type",
+        "description": "In order to be able to access balenaOS versions of a private device type, you need to provide your bearer token. The `deviceType` parameter is required. Even though this query only selects the `raw_version` field, you can additionally specify any of the fields found in the `release` resource.",
```

Comment:
Typo in your suggestion @myarmolinsky

Reply:
I feel that's a bit redundant tbh @myarmolinsky , since the query either selects it doesn't, it's not a default function parameter of a function for example. Lets leave Chris' perfection intact :)

Reply:
> Typo in your suggestion @myarmolinsky 

Thank you :P